src/exposurelog/routers/find_exposures.py

```diff
@@ -142,21 +187,28 @@ async def find_exposures(
         )
         conditions.append("exposure.timespan OVERLAPS date_span")
 
-    # If order_by does not include "id" then append it, to make the order
```

Comment:
Isn't this comment still relevant? The code is adding "id" if it's not there.

Reply:
Yes. I think I deleted it because it was in the wrong place, but in fact it is still relevant, it just needs to be a few lines later.